lib/index.js

```diff
@@ -245,5 +245,13 @@ module.exports = config => {
         return super.validate(req, res, callback);
       }
     }
+
+    getBackLink(req, res) {
+      let getBackLink = super.getBackLink(req, res);
```

Comment:
`getBacklink` is confusing here and would fail lint in some contexts because it's the method name, the parent method name and the temporary var name. Why don't you change the var name to `backLink`?

Reply:
good point 👍

Reply:
I have now amended, thanks